docs/DEPLOYMENT.md

```diff
@@ -8,34 +8,59 @@ Note: we auto deploy the master branch to our *development environment*: [http:/
 
 ## Tag Release ##
 
-On the Monday after the milestone closes, Tag the release.
+This will happen on Friday at the end of sprint.
 
 1. https://github.com/mozilla/testpilot/releases/new
 2. Tag Version: YYYY-MM-DD (append -N if more than one release is tagged on a given day: 2016-04-08-1)
 3. Release Title: YYYY-MM-DD
 4. Click `Publish`
 5. Wait for CircleCI to confirm that the release has passed ([tags uploaded to container](https://hub.docker.com/r/mozilla/testpilot/tags/)).
 
+Please be as detailed as possible in the release notes.
```

Comment:
It's probably good to either point to an example or list the details that should be included. The more explicit the docs are, the easier it'll be for people to take vacations and hand off chores ^_^